package.json

```diff
@@ -7,23 +7,29 @@
   },
   "workspaces": [
     "common/*",
+    "docker-integrations/*",
     "integrations/*",
+    "proxies/*",
     "tools/*"
   ],
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^5.42.1",
     "@typescript-eslint/parser": "^5.42.1",
+    "@protobuf-ts/protoc": "2.9.1",
     "eslint": "^8.27.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-simple-import-sort": "^8.0.0",
     "lerna": "^7.1.1",
     "nx": "^14.4.3",
     "prettier": "^2.3.0",
-    "rimraf": "^3.0.2"
+    "rimraf": "^3.0.2",
+    "protobufjs": "^6.8.6",
+    "ts-proto": "^1.159.2"
   },
   "scripts": {
     "setup": "lerna clean && yarn install && lerna run build",
+    "docker-setup": "yarn nx reset && yarn install && lerna run build",
```

Comment:
@kchrist-rocketfueldev Can we rename this to `build:docker` to follow the same naming convention where `build` is the namespace (or pipeline stage) and `docker` is the target which means you build the docker image.

It follows the same practice as `build:binaries`.